utils/check_repo.py

```diff
@@ -116,6 +116,11 @@
     "XLMForQuestionAnswering",
     "XLNetForQuestionAnswering",
     "SeparableConv1D",
+    "VisualBertForRegionToPhraseAlignment",
+    "VisualBertForVisualReasoning",
+    "VisualBertForQuestionAnswering",
+    "VisualBertForMultipleChoice",
```

Comment:
@sgugger @patil-suraj and I discussed this. It's not the same as regular QA/MultipleChoice, since it needs visual inputs, hence I removed it from auto-class.